TcgPocket/App/src/components/buttons/DeleteButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { ButtonProps, MantineTheme, Button, CSSObject } from '@mantine/core';
+import { ButtonHTMLAttributes } from 'react';
+
+type DeleteButtonProps = ButtonProps & ButtonHTMLAttributes<HTMLButtonElement>;
+
+const buttonStyling = (theme: MantineTheme): CSSObject => ({
+  backgroundColor: theme.colors.red,
+  color: theme.white,
```

Comment:
```suggestion
  backgroundColor: theme.colors.red,
  '&:hover': {
    backgroundColor: 'darkred',
  },
  color: theme.white,
```

I would recommend something like this for the Delete button. At the moment, when you hover over it, it turns blue.